public/static/app/perf/perf.js

```diff
@@ -631,27 +658,36 @@ mciModule.controller('PerfController', function PerfController(
         // Populate the trend data
         var chartDataQ = $http.get("/plugin/json/history/" + $scope.task.id + "/perf").then(
           function(resp) {
-            $scope.trendSamples = new TrendSamples(resp.data);
-            $scope.metricSelect.options = [$scope.metricSelect.default].concat(
-              _.map(
-                _.without($scope.trendSamples.metrics, $scope.metricSelect.default.key),
-                function(d) { return {key: d, name: d} }
+            getRejectedPointsQ.then(function(rejects){
+              let data = resp.data;
+              if(rejects.length){
+                data = _.reject(resp.data, function(doc) {
+                  // return  $scope.task.id != doc.task_id && _.contains(rejects, doc.task_id);
```

Comment:
If we always want to display the data for the current task then this is the logic to apply. However, I think that if we want to show rejected data then we should add some toggle to allow this functionality.